korman/operators/op_mesh.py

```diff
@@ -0,0 +1,369 @@
+#    This file is part of Korman.
+#
+#    Korman is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    Korman is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with Korman.  If not, see <http://www.gnu.org/licenses/>.
+
+import bpy
+import bmesh
+import math
+import mathutils
+
+class PlasmaMeshCreator:
```

Comment:
I think the typical naming convention is `PlasmaMeshOperator` IIRC

Reply:
I think I named this before I figured out where it ultimately ended up.